guide/english/css/before-selector/index.md

```diff
@@ -1,68 +1,61 @@
 
 ```
 
-In the example above we are prepending a grey border before every paragraph element on a page and we are also prepending the words comment in blue before every span element with the class comment.
+In the example above we are prepending a grey border before every paragraph element on a page and we are also prepending the word "Comment: " in blue before every `span` element with the class name `comment`.
 
-> You can check out this demo here https://jsfiddle.net/398by400/
+> You can check out this demo here https://jsfiddle.net/kxrdswtm/
 
-#### Definition and usage
-`::before` is one of the CSS pseudo-elements selectors, which are used to style specified parts of an element. In this case, we can insert content before some HTML element from CSS. Although we will see the content in the page, it is not part of the DOM, what means that we can't manipulate it from Javascript. One trick to solve this handicap: passing the content with a data attribute and use jQuery to manipulate it. Here is an example of use:
+## Definition and Usage
+`::before` is one of the CSS pseudo-elements selectors, which are used to style specified parts of an element. In this case, we can insert content before some HTML element from CSS. Although we will see the content in the page, it is not part of the Document Object Model (DOM), which means that we cannot manipulate it from JavaScript.
 
 ```css
 p::before {
-  content: "Hello ";
+  content: "Hello, ";
 }
 ```
 
 ```html
-<p>world!!</p>
+<p>World!</p>
 ```
 
-This will show `Hello world!!` in the page.
-
-Not only strings, also images, counters or even nothing ("", useful for clearfix) can be inserted into the `content` attribute, but <strong>not HTML</strong>. There are a good number of cool things that can be made using ```::before``` and ```after``` in a creative way. You can take a look in the next link if you are curious: <a href='https://www.w3schools.com/css/css_pseudo_elements.asp' target='_blank' rel='nofollow'>A Whole Bunch of Amazing Stuff Pseudo Elements Can Do</a>
-
-#### Single-colon vs. Double-colon
-There's a bit of discussion about the right way of using pseudo-elements: old style single-colon (```:before```), used in CSS specifications 1 and 2, versus CSS3 recommendation, double-colon (```::before```), mainly to <em>"establish a discrimination between pseudo-classes and pseudo-elements"</em>. But for compatibility reasons, single-colon is still accepted. Talking about compatibility, IE8 supports the single-colon notation only. 
+This will display `Hello, World!` on the page.
 
-#### More Information:
+String, images, counters, or even an empty string ("", useful for `clearfix`) can be inserted into the `content` attribute.  HTML cannot be inserted.
 
-<a href='https://www.w3schools.com/css/css_pseudo_elements.asp' target='_blank' rel='nofollow'>W3Schools CSS Pseudo-elements</a>
+[See A Whole Bunch of Amazing Stuff Pseudo Elements Can Do!](https://www.w3schools.com/css/css_pseudo_elements.asp)
```

Comment:
```suggestion
[See A Whole Bunch of Amazing Stuff Pseudo Elements Can Do!](https://css-tricks.com/pseudo-element-roundup/)
```
I feel this is better and shows more variety

Reply:
I find both of those links to be valid.  My goal was not really to modify the content so much as clean everything up - if you'd like to submit a pull request to change that link please feel free!